lib/simplifyExpression/basicsSearch/index.js

```diff
@@ -6,49 +6,42 @@
 const Node = require('../../node');
 const TreeSearch = require('../../TreeSearch');
 
-// const rearrangeCoefficient = require('./rearrangeCoefficient');
-// const reduceExponentByZero = require('./reduceExponentByZero');
+const rearrangeCoefficient = require('./rearrangeCoefficient');
+const reduceExponentByZero = require('./reduceExponentByZero');
 const reduceMultiplicationByZero = require('./reduceMultiplicationByZero');
-// const reduceZeroDividedByAnything = require('./reduceZeroDividedByAnything');
+const reduceZeroDividedByAnything = require('./reduceZeroDividedByAnything');
 const removeAdditionOfZero = require('./removeAdditionOfZero');
-// const removeDivisionByOne = require('./removeDivisionByOne');
```

Comment:
The reason why I commented these out as opposed to deleting is that I knew for sure that these tests were coming back for sure.